src/python/pants/backend/python/typecheck/mypy/rules_integration_test.py

```diff
@@ -398,63 +396,6 @@ def add(x: int, y: int) -> str:
     assert f"{PACKAGE}/math/add.py:5" in result[0].stdout
 
 
-@skip_unless_python27_present
-def test_works_with_python27(rule_runner: RuleRunner) -> None:
```

Comment:
I don't think we want to delete these tests, but rather have them continue to run against the earlier version of mypy.